homeassistant/components/tts/__init__.py

```diff
@@ -245,12 +248,14 @@ def async_register_engine(self, engine, provider, config):
         self.providers[engine] = provider
 
     @asyncio.coroutine
-    def async_get_url(self, engine, message, cache=None):
+    def async_get_url(self, engine, message, cache=None, language=None):
         """Get URL for play message.
 
         This method is a coroutine.
         """
-        msg_hash = hashlib.sha1(bytes(message, 'utf-8')).hexdigest()
+        msg_hash = hashlib.sha1(bytes(
```

Comment:
What's the reason for the whole {}_{} key separation? Why not have a single hash of everything?